.ci/magician/cmd/check_cassettes.go

```diff
@@ -56,6 +55,8 @@ var checkCassettesCmd = &cobra.Command{
 			env[ev] = val
 		}
 
+		env["GITHUB_TOKEN_DOWNSTREAMS"] = githubTokenOrFallback("GITHUB_TOKEN_DOWNSTREAMS")
```

Comment:
Could you use githubTokenOrFallback("GITHUB_TOKEN_DOWNSTREAMS") instead of accessing env later in this function?